integration-cli/docker_api_stats_test.go

```diff
@@ -231,6 +231,9 @@ func (s *DockerSuite) TestApiStatsContainerNotFound(c *check.C) {
 func (s *DockerSuite) TestApiStatsContainerGetMemoryLimit(c *check.C) {
 	testRequires(c, DaemonIsLinux)
```

Comment:
Hum if it's defined on a `_unix.go` file then it will fail on windows platform if you use it in this file (there might be/need a `docker_api_stats_unix_test.go` with a build tag then…).